catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -309,6 +319,11 @@ protected static String getQueryMetricsLog(Map<String, Serializable> properties)
     properties.entrySet().stream()
         .filter(e -> e.getKey() != null && e.getKey().startsWith(QM_TIMED_SOURCE))
         .forEach(e -> sourceDurationMap.put(e.getKey().split(QM_TIMED_SOURCE)[1], e.getValue()));
+    Serializable cacheSourceValue = sourceDurationMap.remove("cache");
+    if (cacheSourceValue != null) {
+      String sourceName = (String) properties.get(CACHE_SOURCES_KEY);
+      sourceDurationMap.put(sourceName == null ? "Unknown Source" : sourceName, cacheSourceValue);
+    }
```

Comment:
❓ ❗  This feels like this should be handled in downstream projects.  In DDF, the source and the cache can be queried together and this will override/replace any information coming from the original source duration statistics.  If a downstream project wants to report the statistics this way, it could do this level of manipulation as a plugin rather than forcing all DDF programs to report the same.  It may be important to know how long the cache query took compared to the original source.  Also, this assumes only a single source was queried in the cache.  That doesn't have to be true so the sources key might be a list of sources, which this code doesn't handle.